t.yaml

```diff
@@ -0,0 +1,8 @@
+Parameters:
+        UnusedParameter:
+                Type: String
+
+
```

Comment:
cfn-lint: W2001 Parameter AnotherOne not used.  <!-- Automatically posted by Lintly -->